packages/utils/src/ts-eslint/Rule.ts

```diff
@@ -36,6 +36,11 @@ interface RuleMetaDataDocs {
   extendsBaseRule?: boolean | string;
 }
 interface RuleMetaData<TMessageIds extends string> {
+  /**
+   * Specifies default options for the rule.
+   * If present, ESLint will recursively merge any user-provided options with these default options.
+   */
```

Comment:
```suggestion
   * @since X.Y.Z
   */
```

Do you know what version this will be supported in?
We like to documenting this so that consumers don't get confused about something not working.

Reply:
https://github.com/eslint/eslint/pull/17656 is still in review (and waiting on me 😬) - so.. unknown yet.

I think this comment will want to add a `@see` to the ESLint docs, too.

Reply:
I think it's a good idea to include this @since and @see. I'll move this to draft until the feature PR is merged.